models/equalizer/fantom/equalizer_fantom_pools.sql

```diff
@@ -7,13 +7,13 @@
     unique_key = ['pool'],
     post_hook='{{ expose_spells(\'["fantom"]\',
                                 "project",
-                                "equalizer_exchange",
+                                "equalizer",
                                 \'["Henrystats"]\') }}'
     )
 }}
 
 SELECT 'fantom' AS blockchain
-, 'equalizer_exchange' AS project
+, 'equalizer' AS project
```

Comment:
can you also add `schema` property in config block of all these models?
then remove schema assignment for equalizer in dbt_project file?

it'll be good to get head start on that for this dex as we do the same for others moving forward